be/src/olap/olap_common.h

```diff
@@ -239,7 +239,7 @@ class Field;
 class WrapperField;
 using KeyRange = std::pair<WrapperField*, WrapperField*>;
 
-static const int GENERAL_DEBUG_COUNT = 4;
+static const int GENERAL_DEBUG_COUNT = 0;
```

Comment:
Why change it to 0?

Reply:
`general_debug_ns` is used for debugging, by default, there is no place to call it. Therefore, `GENERAL_DEBUG_COUNT` is set to 0, reducing unnecessary overhead.
When developers need to debug, they can change it according to their own needs.